### ISSUE TITLE
'required' parameters information not being read by screen reader

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3528]
### Describe the Bug with repro steps

1. Create a flow in DesignerStandalone (nx serve) and open nvda screen reader.
2. Add an action with required fields, for example 'Append to string variable'
3. Keyboard tab over parameter fields
4. Observe that screen reader does not read aloud 'required' for the parameters.

A related issue is that '<label> is required' text should be read aloud by screen reader.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/724a552c-7b12-43f1-9984-234484a33519)

### Browser

Edge

### Additional context

Related ADO bug: https://msazure.visualstudio.com/One/_workitems/edit/11013773

AB#25598851
